### DISCUSSION TITLE
Is there a step by step guide for eclipse-mosquitto? :)

### DISCUSSION THREAD

**Discussion by FollowTheWizard** [simonsickle/unraid-templates#6] [Q&A / Support]
i am running in some issues that the config, that i manually created cannot been opened to write same as the password file for users.
folder is owned by 1883:1883 and the files are rw-rw-rw.
i am a tiny bit stuck :(

**Comment by simonsickle** [comment]
Try to run a chown to nobody:users and see if that helps? You can also check the logs on the container for other clues and post them here 

Right now, since I only setup the template to use the official image, the only guide there is is [the readme on the docker page](https://hub.docker.com/_/eclipse-mosquitto)
